src/utils/api/transactions.js

```diff
@@ -30,10 +30,12 @@ export const getTransactions = ({
 export const getSingleTransaction = ({ activePeer, id }) => activePeer.transactions.get({ id });
 
 export const unconfirmedTransactions = (activePeer, address, limit = 20, offset = 0, sort = 'timestamp:desc') =>
-  activePeer.node.getTransactions('unconfirmed', {
-    senderId: address,
-    recipientId: address,
-    limit,
-    offset,
-    sort,
+  new Promise(() => {
```

Comment:
Why are you wrapping it with `new Promise`? `activePeer.node.getTransactions` already returns a promise. 
The pending transactions stay in the list, but not sure if it's related to this.

Reply:
I had a problem in one spot not sure now which one, I would have to investigate it and let you know maybe it is already fixed with a current changes.

Reply:
Wrapping the promise here means that it never gets resolved, so you'll prevent if there was some error, but some feature will be broken.